dubbo-config/dubbo-config-spring/src/main/java/org/apache/dubbo/config/spring/beans/factory/annotation/ReferenceAnnotationBeanPostProcessor.java

```diff
@@ -149,13 +151,27 @@ private InvocationHandler buildInvocationHandler(String referencedBeanName, Refe
 
         private Object bean;
 
+        private final Logger logger = LoggerFactory.getLogger(ReferenceBeanInvocationHandler.class);
+
+
         private ReferenceBeanInvocationHandler(ReferenceBean referenceBean) {
             this.referenceBean = referenceBean;
         }
 
         @Override
         public Object invoke(Object proxy, Method method, Object[] args) throws Throwable {
-            return method.invoke(bean, args);
+            try {
+                return method.invoke(bean, args);
+            } catch (Throwable e) {
+                logger.error("Convert RpcException to real exception");
```

Comment:
i think better to log throwable also e.g. logger.error("Convert RpcException to real exception",t);